flaml/automl.py

```diff
@@ -624,6 +655,13 @@ def _prepare_data(self,
                 f"requires input data with at least {n_splits*2} examples.")
             self._state.kf = RepeatedStratifiedKFold(
                 n_splits=n_splits, n_repeats=1, random_state=RANDOM_SEED)
+        elif self._split_type == "time":

```

Comment:
Combine branch with the first one. 

```python
if self._split_type == 'time':
    if eval_method == 'holdout': 
        # do stuff
    else:
        # do stuff
```

Reply:
I am going to leave it the way it is since it keeps the original structure of the code and keeps everything together: lines 548-633 are for holdout, lines 641-669 are for k-fold validation